quickwit/quickwit-indexing/src/models/indexing_directory.rs

```diff
@@ -55,7 +53,6 @@ pub struct IndexingDirectory {
 
 struct InnerIndexingDirectory {
     root: Root,
-    pub cache_directory: PathBuf,
```

Comment:
We can remove the indexing directory indirection in a follow-up PR since it no longer holds two directories but one.

Reply:
Created #2128 accordingly